frontend-legacy/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@bcgov/bc-sans": "^2.1.0",
-    "@bcgov/bootstrap-theme": "https://github.com/bcgov/bootstrap-theme/releases/download/v1.1.1/bcgov-bootstrap-theme-1.1.1.tgz",
+    "@bcgov/bootstrap-v5-theme": "^1.0.1",
```

Comment:
Is this the same package? looks like 1.0.1 is the latest on NPM but it was previously using 1.1.1?